governance/votes/Executive vote - August 14, 2020.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Template - [Executive Vote] Modify Multiple System Parameters
+
+---
+### Reduce the Base Rate
+
+As per the weekly Base Rate[poll](https://vote.makerdao.com/polling-proposal/qmpvqs7gycpjdcpwbye4twvwxuhf4ax2tvt6i1pto5qd2p) the [Base Rate](https://forum.makerdao.com/t/discussion-change-the-stability-fee-structure/2258) meta-parameter will be reduced from -4% to -6% if this proposal passes.
```

Comment:
Link should be: https://vote.makerdao.com/polling-proposal/qmeftwttrdrtiemdoyrmtjnssmlnbdsag2ryxjoycb9l47

Reply:
Fixed.